cli/cli.ts

```diff
@@ -2855,6 +2855,7 @@ export function serveAsync(parsed: commandParser.ParsedCommand) {
             browser: parsed.flags["browser"] as string,
             serial: !parsed.flags["noSerial"] && !globalConfig.noSerial,
             noauth: parsed.flags["noauth"] as boolean || false,
+            backport: parsed.flags["backport"] as number || 0,
```

Comment:
Can you give an example of how the backport flag works? I read in your description that it allows for the simulator extensions to talk to the locally running backend, but I'm confused how you can just cast the flag as a number here. It looks like `port` and `wsport` do the same thing.. I guess I'm just not familiar enough with the cli 😅 . Is the format something like `--backport=3000`?